### PR TITLE
Update/moro vol npcs

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: none

* **Server Mode**: renewal

* **Description of Pull Request**: 

- [ ] Implement a command for questinfo to display the icon when the variable condition are met

### CODE REVIEW

File: npc/re/quests/quests_14_3_bis.txt [hunk 1]
```diff
@@ -0,0 +1,2083 @@
...
+			mes "make those Demon runts so strong.";
+			next;
+			mes "[Avant]";
+			mes "I want you to";
+			mes "get rid of them and bring me Mana Crystallines.";
+			mes "Don't try to keep them from me: they're not useful for you.";
+			next;
+			mes "[Avant]";
+			mes "Three Reincarnations, thus three Mana Crystallines.";
+			mes "Bring them at least for Cardui of Eclage.";
```
Comment (zackdreaver) [inline npc/re/quests/quests_14_3_bis.txt]: For translation consistency, can we use Kardui instead?

**Comment by Questune09** [comment]
after killing fire pit they dont spawn fire bug i think on official after killing fire pit they spawn 2pcs fire bug?>

**Comment by Atemo** [comment]
You probably didn't update your master branch
Relevant commit https://github.com/rathena/rathena/commit/6e0187bc2b7d470a84ce6dddfb71f80be8f56b53

**Comment by ecdarreola** [comment]
Works fine for me. Didn't encounter @Questune09 's bug.

File: npc/re/mobs/dungeons/moro_vol.txt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+moro_vol	monster	Incarnation of Morocc	1920,15
+moro_vol	monster	Incarnation of Morocc	1921,15
+moro_vol	monster	Fire Condor	3020,30
+moro_vol	monster	Fire Sandman	3021,30
+moro_vol	monster	Fire Frilldora	3022,30
+moro_vol	monster	Fire Golem	3023,30
+moro_vol	monster	Fire Bug	3027,10
+moro_vol	monster	Blazing Morocc Reincarnation	3039,1
+moro_vol	monster	Distorted Morocc Reincarnation	3040,1
+moro_vol	monster	Freezing Morocc Reincarnation	3041,1
```
Comment (Questune09) [inline npc/re/mobs/dungeons/moro_vol.txt]: i think this are wrong definition

it should be something like this

moro_vol,0,0	monster	Fire Pit	3026,10,5000

Comment (Atemo) [reply npc/re/mobs/dungeons/moro_vol.txt]: In commit https://github.com/rathena/rathena/pull/3002/commits/0d276f864da9030c3a9969a07ea64b6f51ae5809 `Adjusted permanent monster spawn to 0 (random spawn) xy by default` : the map name alone define a random spawn.
The respawn timer is optionnal. Monster never respawn faster than 5 seconds : https://github.com/rathena/rathena/blob/master/src/map/mob.cpp#L1082

**Pull request merged**